src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/personalrelationships/ContactPersonMigrationMessageListener.kt

```diff
@@ -20,7 +20,7 @@ import java.util.concurrent.CompletableFuture
 class ContactPersonMigrationMessageListener(
```

Comment:
convert this to use PrisonerRestrictions rather than deleting and created from new. This will be a one shot migration so happy to cheat a little bit here